### PR TITLE
Update docs version

### PR DESCRIPTION
**Pull request by guarin**
Update docs version and link copyright in docs footer to lightly website

Closes #618

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 author = 'Philipp Wirth, Igor Susmelj'
 
 # The full version, including alpha/beta/rc tags
-release = '1.0.0'
+release = '1.2.0'
```
Comment (philippmwirth) [inline docs/source/conf.py:27]: You could try to get this on from `../lightly/__init__.py`

Comment (guarin) [reply docs/source/conf.py]: Works :)

File: docs/source/conf.py [hunk 2]
```diff
@@ -15,16 +15,19 @@
 sys.path.insert(0, os.path.abspath('../..'))
 
 import sphinx_rtd_theme
+import lightly
```
Comment (philippmwirth) [inline docs/source/conf.py:18]: are you sure this doesn't import an installed version of lightly? which might mess up the version... so the procedure would be 
1. install lightly from source
2. release the docs

File: docs/source/conf.py [hunk 3]
```diff
@@ -15,16 +15,19 @@
...
 project = 'lightly'
-copyright = '2020, Lightly AG'
+copyright_year = '2020'
+copyright = "Lightly AG"
+website_url = 'https://www.lightly.ai/'
 author = 'Philipp Wirth, Igor Susmelj'
 
 # The full version, including alpha/beta/rc tags
-release = '1.0.0'
+release = lightly.__version__
```
Comment (japrescott) [inline docs/source/conf.py:30]: very nice

**Review by japrescott — COMMENTED** [review]
good work with the footer!

File: docs/source/conf.py (hunk 2 above)
Comment (guarin) [reply docs/source/conf.py:18]: The `sys.path.insert(0, os.path.abspath('../..'))` makes sure that lightly is imported from the local directory and not the installed package.

Comment (philippmwirth) [reply docs/source/conf.py:18]: ah cool, didn't see that

**Review by philippmwirth — APPROVED** [review]
Looks good, did you try it out and build the docs?

**Comment by guarin** [comment]
Yes, did not completely rebuild docs but the changes work as expected. If I change the lightly version without reinstalling the package it successfully updates the version displayed in the docs.

**Pull request merged**